dockerfiles/Dockerfile.arm32v7

```diff
@@ -67,21 +68,24 @@ COPY conf/fluent-bit.conf \
      conf/plugins.conf \
      /fluent-bit/etc/
 
-FROM arm32v7/debian:buster-slim
+FROM arm32v7/debian:bullseye-slim
+LABEL description="Fluent Bit docker image" vendor="Fluent Organization" version="1.9.0" author="Eduardo Silva <eduardo@calyptia.com>"
```

Comment:
can we get the version from a variable?

Reply:
Yeah, I was just keeping changes minimal from what we do currently. It would require a new build arg but it can be done.

Reply:
There also seems to be a script that does the updates, the main thing to change is here:
```
# Fluent Bit version
ENV FLB_MAJOR 1
ENV FLB_MINOR 9
ENV FLB_PATCH 0
ENV FLB_VERSION 1.9.0
```

We should provide `FLB_VERSION` as a build argument defaulted:
```
ARG FLB_VERSION=1.9.0
ENV FLB_VERSION=${FLB_VERSION}
```
Anywhere we use a hard-coded version we can then just use this. It may be better to set all those variables from build arguments.
I'd suggest a separate change for this though?

Reply:
Yes, lets go in a separate change.